openml-datarobot/pom.xml

```diff
@@ -87,6 +87,17 @@
         </dependency>
     </dependencies>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.feedzai</groupId>
+                <artifactId>openml-api</artifactId>
+                <version>${openml-api.version}</version>
+                <scope>provided</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
It's not clear to me why you've added this (and also on h2o) to depedencyManagement. DependencyManagement is used to "declare" (but not necessarily use) the dependencies.

Reply:
Sorry, forgot to remove this. I was using this to solve an issue I had with IDEA where it could resolve the right version of openml-api from the parent pom.